src/args.rs

```diff
@@ -185,7 +183,10 @@ mod tests {
     }
 
     #[rstest_parametrize(
-        auth_string, username, password, encrypt,
+        auth_string,
+        username,
+        password,
+        encrypt,
```

Comment:
Can we keep these like they were? Makes it clearer.

Reply:
We can yes. I'll have to tell rustfmt not to touch this part and it should be fine :)